common/src/main/java/org/apache/celeborn/common/client/MasterNotLeaderException.java

```diff
@@ -19,22 +19,27 @@
 
 import java.io.IOException;
 
+import javax.annotation.Nullable;
```

Comment:
seems we don't use it in other places

Reply:
@pan3793, the invoker of [HAHelper.java#L44](https://github.com/apache/incubator-celeborn/blob/main/master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HAHelper.java#L44) will pass null cause, therefore this just adds the annotation to mark this parameter could be nullable.